### PR TITLE
Franklin onboarding journey tutorial code changes 

### PR DESCRIPTION
**Pull request by varunmitra**
Code changes for Franklin onboarding journey.
Videos hosted on Dynamic Media
Blocks added:
FAQ
Video Text

Test page: https://main--helix-website--kunwarsaluja.hlx.page/drafts/cashub/tutorial-test

### CODE REVIEW

**Comment by varunmitra** [comment]
@rofe can you please review and approve this PR? This is for the Franklin onboarding tutorial videos created by CAS Hub.
Tagging @3vil3mpir3, @kunwarsaluja

File: blocks/videotext/videotext.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.videocontent video {
+  width: 100%;
+}
+
+.videotext .text {
+  font-size: var(--type-body-s-size);
+  padding-left: 10px;
+}
```
Comment (rofe) [inline blocks/videotext/videotext.css:23]: Can you turn the CSS around so it is mobile first and use 600, 900 and 1200 breakpoints? See other blocks for examples:
1. default rules for mobile
2. override rules for tablet and desktop using media queries with `min-width:...`

File: styles/styles.css [hunk 2]
```diff
@@ -642,4 +642,10 @@ body.docs-template main .section.dark .default-content-wrapper {
   border-color: var(--spectrum-blue);
 }
 
-
+main .separator
```
Comment (rofe) [inline styles/styles.css:645]: Make it clear that this is a `.section` rule:
```suggestion
main .section.separator
```

**Review by rofe — CHANGES_REQUESTED** [review]
Please re-add the `faq` block, it is used in production ;)
https://www.hlx.live/docs/faq

**Comment by varunmitra** [comment]
Closing this PR in favor of https://github.com/adobe/helix-website/pull/144

**Pull request closed without merging**